### PR TITLE
Migrate PublishTestResultsV1 to Node 10 

### PR DESCRIPTION
**Pull request by ghost**
**Task name**: PublishTestResultsV1

**Description**: this pull request provides updating the PublishTestResultsV1 task and its migration to Node 10

**Documentation changes required:** (Y/N) N

**Added unit tests:** (Y/N) N

**Attached related issue:** (Y/N) N

**Checklist**:
- [x] Task version was bumped - please check [instruction](https://github.com/microsoft/azure-pipelines-tasks/tree/master/docs/taskversionbumping.md) how to do it
- [x] Checked that applied changes work as expected

This task has been built and uploaded (via the tfx-cli tool) into the Azure DevOps organization and tested end-to-end running tests with different test runners and uploaded their results

### CODE REVIEW

File: Tasks/PublishTestResultsV1/task.json [hunk 1]
```diff
@@ -13,7 +13,7 @@
     "author": "Microsoft Corporation",
```
Comment (anatolybolshakov) [inline Tasks/PublishTestResultsV1/task.json:13]: Should we bump minimum agent version also?

**Comment by anatolybolshakov** [comment]
@tatyana-kostromskaya please check ci failures as well

**Review by anatolybolshakov — APPROVED** [review]

**Review by DenisRumyantsev — APPROVED** [review]
LGTM, thanks

**Comment by tatyana-kostromskaya** [comment]
cc @geekzter for visibility

**Comment by DenisRumyantsev** [comment]
I am closing this PR. The re-created PR is here —
* https://github.com/microsoft/azure-pipelines-tasks/pull/16664

**Pull request closed without merging**